templates/cjs/typescript_gapic/src/$version/$service_client.ts.njk

```diff
@@ -389,7 +389,10 @@ export class {{ service.name }}Client {
     // Put together the default options sent with requests.
     this._defaults = this._gaxGrpc.constructSettings(
         '{{ api.naming.protoPackage }}.{{ service.name }}', gapicConfig as gax.ClientConfig,
-        opts.clientConfig || {}, {'x-goog-api-client': clientHeader.join(' ')});
+        opts.clientConfig || {}, {'x-goog-api-client': clientHeader.join(' ')
+{%- if service.apiVersion -%}
+       , 'x-goog-api-version': '{{ service.apiVersion }}'
```

Comment:
It seems like this change is not covered by baselines, so if it stops working for whatever reason, we won't be able to catch it. Should we add the corresponding annotation to at least one baseline service?